VachaarBack/settings.py

```
@@ -192,8 +193,8 @@
 # JWT SETTINGS
 SIMPLE_JWT = {
     "ALGORITHM": "RS256",
-    "SIGNING_KEY": env("PRIVATE_KEY"),
-    "VERIFYING_KEY": env("PUBLIC_KEY"),
+    "SIGNING_KEY": open("private.key", "rb").read(),
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)